_health-care/_js/components/Nav.jsx

```diff
@@ -12,84 +12,84 @@ class Nav extends React.Component {
     // TODO(akainic): change this check once the alias for introduction has been changed
     return (
       <ol className="process">
-        <li className="step one wow fadeIn animated">
```

Comment:
Dry this up by defining a `const subnavStyles="step one wow fadeIn animated";` and then using that into each className section?